macos/ONLYOFFICE/update/updates/ONLYOFFICE/changes/8.1.0/changes.ru.html

```diff
@@ -81,56 +81,90 @@
                 color: var(--color-gray-600);
             }
         }
-     </style>
+    </style>
 </head>
 <body>
     <div class="version">
         <h1>ONLYOFFICE 8.1.0<span class="releasedate"> - {{DATE}}</span></h1>
 
-        <h2>Новые функции</h2>
+        <h2>New Features</h2>
 
-        <h4>Все редакторы</h4>
+        <h4>All Editors</h4>
         <ul>
-            <li>TODO</li>
+            <li>Switching to the creating forms in the <code>PDF</code> format</li>
+            <li>Added the ability to disable the 'Connect to cloud' section via the <code>--lock-portals</code> key</li>
```

Comment:
<li>Added the ability to disable the <code>Connect to cloud</code> section via the <code>--lock-portals</code> key</li>